modules/apps/fragment/fragment-impl/src/main/java/com/liferay/fragment/internal/renderer/FragmentEntryFragmentRenderer.java

```diff
@@ -331,10 +345,16 @@ private String _renderFragmentEntryLink(
 					fragmentEntryLink.getEditableValues());
 		}
 
-		content = _renderFragmentEntry(
-			fragmentEntryLink.getFragmentEntryId(), css, html,
-			fragmentEntryLink.getJs(), configurationJSONObject.toString(),
-			fragmentEntryLink.getNamespace(), httpServletRequest);
+		if (FragmentTypeUtil.isReactFragment(fragmentEntryLink)) {
+			content = _renderReactFragmentEntryLink(
```

Comment:
I think this should probably at the very least also consider:
- `css`: render it if provided
- `html`: render it if provided (could be used for ssr or placeholder content)
- `configurationJSONObject`: should be passed as props
- `namespace`: if this is a namespace per fragmentEntryLink, then should be also passed as props or something similar to what it does right now to the JS method

Reply:
Makes sense, though it should be clearly defined what to expect and do with each value.